Makefile

```diff
@@ -201,18 +201,15 @@ helm-docs: $(HELM_DOCS) ## Use helm-docs to generate documentation from helm cha
 		-t ../../../Documentation/Helm-Charts/ceph-cluster-chart.gotmpl.md \
 		-t ../../../Documentation/Helm-Charts/_templates.gotmpl
 
-check-helm-docs:
+check.helm-docs:
```

Comment:
Do we wanna do `check.helm.docs`

Reply:
@parth-gr  wrote:
> Do we wanna do `check.helm.docs`

we could, but that would mostly make sense if there were more `check.helm.foo` targets

Reply:
I think we can come to a consistent pattern of creating make cmds,
I know what you mean above, but pattern should be common(that means sont use - and always use .)

Reply:
But I am fine with the above change to, for now, I am just looking for a more robust future perspective